hack/check-marker.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
Call this script `run-marker.sh` or `check-markdown.sh` instead, since we're not really "checking" marker itself.

Reply:
Done